courtspecs-migrations/migrate-mongo-config.js

```diff
@@ -0,0 +1,24 @@
+const config = {
+  mongodb: {
+    url: process.env.DATA_MIGRATION_URL,
+
+    databaseName: process.env.DATABASE_NAME,
```

Comment:
when I run this script,  process.env.DATA_MIGRATION_URL, and process.env.DATABASE_NAME  are undefined.